config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,25 @@
+CarrierWave.configure do |config|
+  if Rails.env.development? || Rails.env.staging?
```

Comment:
Good job separating staging and production, however development goes usually locally and staging serves as testing for production stuff.